build-system/tasks/compile.js

```diff
@@ -104,7 +107,17 @@ function compile(entryModuleFilename, outputDir,
       // Don't include tests.
       '!**_test.js',
       '!**/test-*.js',
-    ])
+    ];
+    // Many files include the polyfills, but we only want to deliver them
+    // once. Since all files automatically wait for the main binary to load
+    // this works fine.
+    if (options.includePolyfills) {
```

Comment:
There are two files that resolve to the module name. One is empty. If we should include polyfills, it removes the empty from the set, otherwise it removes the actual file.